### ISSUE TITLE
Smooch tries to update the user after logout

### ISSUE THREAD

**Issue by emilecantin** [smooch/smooch-js#396]
Smooch tries to call PUT https://api.smooch.io/v1/appusers/<USER_EMAIL> with an empty body after calling `Smooch.logout();`, but the server returns a 401 (as it should).

Is there a reason for this? It puts some annoying errors in my app's console, which I'd like to avoid.

Here is an extract from my console log. The first line is printed by a log statement right above the call to `Smooch.logout()`.

```
SmoochWrapper Smooch.logout +1ms
{...}
PUT https://api.smooch.io/v1/appusers/ecantin%40tyl.io 401 (Unauthorized)
http.js:45 Uncaught (in promise) Error: Unauthorized(…)
```

**Comment by jpjoyal** [comment]
Thanks for the report @emilecantin. This indeed seems like a bug, not reason to call the server on logout. I'll add that to our bug queue. Looks like the impact is not too high right ?

**Comment by emilecantin** [comment]
No, we only see an error in the console, nothing user-facing.

**Comment by mlmassey** [comment]
Also seeing the same issue.  From your experience, are the console errors benign, and no other issues come out of this?

**Comment by mspensieri** [comment]
An attempt was made to fix this in [v3.10.0](https://github.com/smooch/smooch-js/releases/tag/3.10.0). Please re-open if you can still reproduce with the latest version
